airbyte-webapp/src/components/connectorBuilder/Builder/InputsView.tsx

```diff
@@ -0,0 +1,303 @@
+
+interface InputInEditing {
+  key: string;
+  definition: JSONSchema7;
+  required: boolean;
+  isNew?: boolean;
+  showDefaultValueField: boolean;
+  type: typeof supportedTypes[number];
+}
+
+function sluggify(str: string) {
+  return str.toLowerCase().replaceAll(/[^a-zA-Z\d]/g, "_");
+}
+
+function newInputInEditing(): InputInEditing {
+  return {
+    key: "",
+    definition: {},
+    required: false,
+    isNew: true,
+    showDefaultValueField: false,
+    type: "string",
+  };
+}
+
+function formInputToInputInEditing({ key, definition, required }: BuilderFormInput): InputInEditing {
+  const supportedType = supportedTypes.find((type) => type === definition.type) || "unknown";
+  return {
+    key,
+    definition,
+    required,
+    isNew: false,
+    showDefaultValueField: Boolean(definition.default),
+    type: supportedType !== "unknown" && definition.enum ? "enum" : supportedType,
+  };
+}
+
+function inputInEditingToFormInput({
+  type,
+  showDefaultValueField,
+  isNew,
+  ...values
+}: InputInEditing): BuilderFormInput {
+  return {
+    ...values,
+    definition: {
+      ...values.definition,
+      type: type === "enum" ? "string" : type === "unknown" ? values.definition.type : type,
+      // only respect the enum values if the user explicitly selected enum as type
+      enum: type === "enum" && values.definition.enum?.length ? values.definition.enum : undefined,
+      default: showDefaultValueField ? values.definition.default : undefined,
+    },
+  };
+}
+
+export const InputsView: React.FC = () => {
+  const { formatMessage } = useIntl();
+  const [inputs, , helpers] = useField<BuilderFormInput[]>("inputs");
+  const [inputInEditing, setInputInEditing] = useState<InputInEditing | undefined>(undefined);
+  const usedKeys = useMemo(() => inputs.value.map((input) => input.key), [inputs.value]);
+  const inputInEditValidation = useMemo(
+    () =>
+      yup.object().shape({
+        // make sure key can only occur once
+        key: yup
+          .string()
+          .required("form.empty.error")
+          .notOneOf(inputInEditing?.isNew ? usedKeys : usedKeys.filter((key) => key !== inputInEditing?.key)),
+        required: yup.bool(),
+        definition: yup.object().shape({
+          title: yup.string().required("form.empty.error"),
+        }),
+      }),
+    [inputInEditing?.isNew, inputInEditing?.key, usedKeys]
+  );
+  return (
+    <BuilderConfigView heading={formatMessage({ id: "connectorBuilder.inputsTitle" })}>
+      <Text centered className={styles.inputsDescription}>
+        <FormattedMessage id="connectorBuilder.inputsDescription" />
+      </Text>
+      {inputs.value.length > 0 && (
+        <Card withPadding className={styles.inputsCard}>
+          <ol className={styles.list}>
+            {inputs.value.map((input) => (
+              <li className={styles.listItem} key={input.key}>
+                <div className={styles.itemLabel}>{input.definition.title || input.key}</div>
```

Comment:
nit: these labels currently look different from the labels in the other views, so we should make them consistent.

This should be achievable by changing this `<div>` to be a `<Label>` instead, so that the same styles are applied.

Reply:
done